app/src/main/java/com/example/tasky/feature_agenda/presentation/event_detail_screen/EventDetailScreen.kt

```diff
@@ -396,14 +388,21 @@ fun EventDetailScreen(
                                                 .clickable {
                                                     navigateTo(
                                                         Screen.PhotoDetailsScreen.route +
-                                                                "/${photo.key}" +
-                                                                "/${Uri.encode(photo.url)}"
+                                                                "/${ when(photo) {
+                                                                    is EventPhoto.Local -> photo.key
+                                                                    is EventPhoto.Remote -> photo.key
+                                                                }}" +
+                                                                "/${Uri.encode(when(photo) {
+                                                                    is EventPhoto.Local -> photo.uri
+                                                                    is EventPhoto.Remote -> photo.url
+                                                                })
+                                                                }"
                                                     )
                                                 }
                                         )
                                     }
                                 }
-                                if(state.photoList.size < 10) {
+                                if(photoList.size < 10 && state.editMode) {
```

Comment:
Naming suggestion: `isInEditMode`
Otherwise it sounds like there are multiple edit modes to choose from